ttnn/cpp/ttnn/operations/normalization/softmax/device/kernels/dataflow/reader_unary_sharded_sm_rm_mask.cpp

```diff
@@ -42,8 +42,9 @@ void kernel_main() {
     uint32_t l1_write_addr = get_write_ptr(cb_attn);
     for (uint32_t w = 0; w < block_wt; w++) {
         uint64_t mask_noc_addr = get_noc_addr(mask_start_tile_id + w, addr_mask);
-        noc_async_read(mask_noc_addr, l1_write_addr, mask_read_tile_face_bytes);
-        mask_noc_addr += mask_read_tile_face_bytes;
+        noc_async_read(mask_noc_addr, l1_write_addr, mask_read_tile_face_bytes * 2);
```

Comment:
Minor: Could you make the declaration of `mask_read_tile_face_bytes` and `mask_read_tile_offset_bytes` constexpr?